docs/standalone.md

```diff
@@ -2,97 +2,292 @@
+if (!validateFoo(fooPass))
+  console.log("ERRORS 1:", validateFoo.errors); //Never reaches here because valid
+
+if (!validateFoo(fooFail))
+  console.log("ERRORS 2:", validateFoo.errors); //Errors array gets logged
 ```
 
+
 ### Requirement at runtime
 
-To run the standalone generated functions, the Ajv package should still be a run-time dependency for most schemas, but generated modules can only depend on code in [runtime](https://github.com/ajv-validator/ajv/tree/master/lib/runtime) folder, so the whole Ajv will not be included in the bundle (or executed) if you require the modules with standalone validation code from your application code.
+One of the main reason for using the standalone mode is to compile the JSON schema into functions at build 
+time and then just consume them during runtime. This prevents the initial compilation at runtime which saves time at runtime. 
+
+It is important to understand that the standalone generated functions still has a dependency on the Ajv. Specifically
+on the code in the [runtime](https://github.com/ajv-validator/ajv/tree/master/lib/runtime) folder of the package. 
+
+Completely isolated validation functions can be generated if desired (won't be for most use cases). Run the generated code
+through a bundler like ES Build to create completely isolated validation functions that can be imported/required 
+without any dependency on Ajv. This is also not needed if your project is already using a bundler.
 
 ## Configuration and limitations
 
 To support standalone code generation:
 
-- Ajv option `source.code` must be set to `true`
-- only `code` and `macro` user-defined keywords are supported (see [User defined keywords](./keywords.md)).
-- when `code` keywords define variables in shared scope using `gen.scopeValue`, they must provide `code` property with the code snippet. See source code of pre-defined keywords for examples in [vocabularies folder](https://github.com/ajv-validator/ajv/blob/master/lib/vocabularies).
-- if formats are used in standalone code, ajv option `code.formats` should contain the code snippet that will evaluate to an object with all used format definitions - it can be a call to `require("...")` with the correct path (relative to the location of saved module):
+- Ajv option `code.source` must be set to `true`
+- Only `code` and `macro` user-defined keywords are supported (see [User defined keywords](./keywords.md)).
+- When `code` keywords define variables in shared scope using `gen.scopeValue`, they must provide `code` property with 
+the code snippet. See source code of pre-defined keywords for examples in 
+[vocabularies folder](https://github.com/ajv-validator/ajv/blob/master/lib/vocabularies).
+- If formats are used in standalone code, ajv option `code.formats` should contain the code snippet that will evaluate 
+to an object with all used format definitions - it can be a call to `require("...")` with the correct path 
+(relative to the location of saved module):
```

Comment:
```suggestion
- If formats are used in standalone code, ajv option `code.formats` should contain the code snippet that will evaluate to an object with all used format definitions - it can be a call to `require("...")` with the correct path (relative to the location of saved module):
```